mypyc/codegen/emitmodule.py

```diff
@@ -993,7 +993,7 @@ def _toposort_visit(name: str) -> None:
             result.append(decl.declaration)
             decl.mark = True
 
-        for name, marked_declaration in marked_declarations.items():
+        for name in marked_declarations.keys():
```

Comment:
Or just iterate directly over the dict:

```suggestion
        for name in marked_declarations:
```